File: cloud_HSM.py
import boto3

def create_hsm():
    client = boto3.client('cloudhsm')

    request = {
        'ClusterId': 'my-cluster-id',
    }

    response = client.create_hsm(**request)

    print(response['HsmId'])

if __name__ == '__main__':
    create_hsm()

File: cloud_HSM.cs
using Amazon.CloudHSM;
using Amazon.CloudHSM.Model;

public class CreateHsm
{
    public static void Main(string[] args)
    {
        var client = new CloudHSMClient();

        var request = new CreateHsmRequest
        {
            ClusterId = "my-cluster-id"
        };

        var response = client.CreateHsm(request);

        Console.WriteLine("HSM created: " + response.HsmId);
    }
}
